### DISCUSSION TITLE
Release quarto 1.5 to CRAN

### DISCUSSION THREAD

**Discussion by jeffwong** [quarto-dev/quarto-cli#10866] [Q&A] [answered]
### Description

I am trying to use a quarto .qmd file to create a pkgdown article for my R package. pkgdown::build_site() says that quarto needs to be on version 1.5. But [CRAN](https://cran.r-project.org/web/packages/quarto/index.html) only has version 1.4. Can the 1.5 version be released on CRAN?

**Comment by mcanouil** [comment, marked as answer]
Please read the description (in particular system requirements) and the documentation of the R package.
https://quarto-dev.github.io/quarto-r/
It's usually helpful to read those components when using R packages or actually any library.

The R package is a wrapper around Quarto CLI. You need to update your Quarto CLI version as described in the R package documentation (which is unrelated to the R package).
